src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
```

Comment:
you can use Math library

Reply:
When I import Math library, MVN Clean package said that it is  "Redundant import from the java.lang package"